llvm/utils/benchmark/src/benchmark_register.h

```diff
@@ -2,6 +2,7 @@
 #define BENCHMARK_REGISTER_H
 
 #include <vector>
+#include <limits>
```

Comment:
Nit: LLVM Coding Standard requires sorting the #include lines in alphabetical order. See the original patch b498303066. Consider cherry-picking the commit preserving original authorship.